pkgs/tools/backup/bacula/default.nix

```diff
@@ -11,6 +11,11 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-AZWgi81PV4rkqc4Nkff4ZzHGNNVrgQU0ci1yGyqe7Lc=";
   };
 
+  postPatch = lib.optionalString (stdenv.isDarwin && stdenv.isAarch64) ''
+    substituteInPlace configure \
+      --replace "10.*)" "*)"
```

Comment:
What does this magic replacement do? Would be nice to have a comment.

Reply:
Added comment.